.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 16
```

Comment:
I was about to say that https://github.com/PhilanthropyDataCommons/pdc/runs/6990785047?check_suite_focus=true#step:3:3 appeared to use the .nodeversion but no, it is what is specified here, I guess.

Reply:
Indeed -- if we had more time I'd spend a cycle checking to see if there's a way to auto-use .node-version but FWIW I haven't seen that pattern in the past (and it actually might be worthwhile to decouple CI from dev env, since in theory we might want to use other versions in CI)